pkg/kubectl/cmd/get.go

```diff
@@ -178,6 +178,10 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 		return err
 	}
 
+	if strings.HasPrefix(cmdNamespace, "namespaces/") {
+		cmdNamespace = strings.Split(cmdNamespace, "/")[1]
```

Comment:
Also, I'm afraid `cmdNamespace = strings.Split(cmdNamespace, "/")[1]` will cause a panic if `cmdNamespace` is specified to `namespaces/`.